dev/system-test/firestore.ts

```diff
@@ -2148,6 +2148,17 @@ describe('WriteBatch class', () => {
       });
   });
 
+  it('has a full stack trace if set() errors', () => {
+    // Use an invalid document name that the backend will reject.
+    const ref = randomCol.doc('__doc__');
+    const batch = firestore.batch();
+    batch.set(ref, {foo: 'a'});
+    return batch.commit().then(() => Promise.reject('commit() should have failed')
```

Comment:
I am glad this is causing a lint failure :) Doesn't look pretty like this. The PR is good to go after `yarn gts fix`.

Reply:
fixed. Also I changed `wrapError()` to use `GoogleError` in order to preserve error codes when re-throwing.